src/packages/@ncigdc/containers/ConsequencesTable.js

```diff
@@ -131,6 +136,7 @@ const ConsequencesTableComponent = compose(
       style={{ width: '100%', minWidth: 450 }}
       data={dataRows}
       prefix={paginationPrefix}
+      entityName={'entries'}
```

Comment:
There are only 4 different consequences on the table though.. I think it's better to keep it ambiguous. Will ask PM